### DISCUSSION TITLE
Adding custom attributes with parameters

### DISCUSSION THREAD

**Discussion by foligo** [elementor/elementor#18141] [Developers Community]
Hey there everyone,
I'm trying to implement plausible.io analytics event tracking on my website which runs elementor pro. Therefore I need to implement custom attributes to links and elements with parameters. 
**Ex.:** data-analytics='"Register", {"props":{"plan":"Navigation","location":"footer"}}' (Docs: [https://plausible.io/docs/custom-event-goals#using-custom-props](https://plausible.io/docs/custom-event-goals#using-custom-props)) 

When I try to insert this within the **custom attributes field either on links or on elements (in advanced section)** elementor parses this in its own way, ex.: 
**Insert:** data-analytics|"kontakt-button", {"props":{"location":"header"}}"
**Parsed:** data-analytics='""kontakt-button"' props=""

Is there any way that elementor does not parse this so randomly? 
Does anybody of you have a solution or experience with his? 

Thanks a lot in advance and best wishes,

**Comment by WebsiteDons** [comment]
Maybe try single quotes or use an escape backslash character

```html
data-analytics|\"kontakt-button\", {\"props\":{\"location\":\"header\"}}\"
```
